src/rustdoc-json-types/lib.rs

```diff
@@ -232,6 +232,7 @@ pub enum ItemEnum {
         type_: Type,
         /// e.g. `const X: usize = 5;`
         default: Option<String>,
+        default_value: Option<String>,
```

Comment:
What's the difference between `default` and `default_value`? Could you document it

Reply:
`default_value` is the *evaluated* form of the default constant expression and `default` is an the *unevaluated* form.
Same story with the existing entries `value` and `expr` for free-standing constants.

Yeah, I should definitely document it if we want to keep it.
However, I'd rather get rid of `default_value` again and put the _evaluated_ expression into `default` entry. Technically also a breaking change I guess but since my hotfix PR #98814 unevaluated exprs have become quite useless (in most cases they are mapped to `_`).
What do you think?
Now that I think about it, #98814 also introduced a breaking change to the JSON output and that was gone unnoticed (no bump in the format version).

Reply:
Then please comment it in the code. :)